src/posts/2012-03-06-webinar-outside-in.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: layouts/post
+title: Webinar&#58; Outside In - Achieving Our Goals by Designing for Diversity
+date: '2012-03-26'
+tags: post
+permalink: posts/2012-03-06-webinar-outside-in.html
+---
+<article class="floe-content floe-news-item">
+                <h2> News </h2>
+                <h3>Webinar: Outside In - Achieving Our Goals by Designing for Diversity</h3>
+                <time class="floe-date" datetime="2012-03-06">March 06, 2012</time>
```

Comment:
Each article can have lines 8 to 11 refactored and put into the `post.html` template

Reply:
Done